plugins/kelper.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kelper
+spec:
+  version: v0.0.19
+  shortDescription: A CLI tool to serve as swiss-army knife for your operations on Kubernetes pods and nodes
```

Comment:
- max 55 chars pls
- it would help to be more concrete. Swiss army knife doesn't mean anything to me tbh :) I can't picture what it does.